### PR TITLE
Add the ability to set a optional timeout

### PR DESCRIPTION
**Pull request by robhenley**

### CODE REVIEW

File: includes/Connector.class.php [hunk 1]
```diff
@@ -7,6 +7,7 @@ class AC_Connector {
 	public $url;
 	public $api_key;
 	public $output = "json";
+	private $timeout = NULL;
```
Comment (cristiangrama) [inline includes/Connector.class.php]: Let's default `timeout` to 30 seconds by using a `const` and assigning it by default to the `timeout` class property.

**Comment by robhenley** [comment]
@cristiangrama Thanks for reviewing.  Why create a class constant and then assign it the class property `timeout` instead of just setting a default value to the `timeout` class property?

**Comment by cristiangrama** [comment]
That is just for readability. You could have it on the property too but I feel that using a constant is more readable. Any of them would work.

```
const DEFAULT_TIMEOUT = 30;

private $timeout = self::DEFAULT_TIMEOUT;
```
vs
```
private $timeout = 30;
```

**Comment by robhenley** [comment]
Thanks for explaining.  I didn't see constants used elsewhere so was curious why.  Updates have been made.

**Pull request merged**